src/models/customs-declarations.js

```diff
@@ -265,3 +265,71 @@ export const mapCustomsDeclarations = ({
+      return decisionCheck.decision
+    }
+  }
+
+  return ''
+}
+
+// Order of these checks matter. It returns the 'worst' case of all the item decisions first.
+const btmsDecisionChecks = [
+  { checkType: 'item', decisionCode: 'E03', decision: 'Data Error - Unexpected data - transit, transhipment or specific warehouse' },
+  { checkType: 'item', decisionCode: 'N01', decision: 'Refuse - Not acceptable' },
+  { checkType: 'item', decisionCode: 'N02', decision: 'Refuse - Destroy' },
+  { checkType: 'item', decisionCode: 'N03', decision: 'Refuse - Transform' },
+  { checkType: 'item', decisionCode: 'N04', decision: 'Refuse - Re-export or re-dispatch' },
+  { checkType: 'item', decisionCode: 'N05', decision: 'Refuse - Use for other purposes' },
+  { checkType: 'item', decisionCode: 'N06', decision: 'Refuse - Refused' },
+  { checkType: 'item', decisionCode: 'N07', decision: 'Refuse - Not acceptable' },
+  { checkType: 'item', decisionCode: 'X00', checkCode: 'H224', decision: 'Refuse - IUU not compliant' },
+  { checkType: 'result', decisionCode: 'E70', decision: 'No match - CHED cannot be found' },
+  { checkType: 'result', decisionCode: 'E71', decision: 'No match - CHED cancelled' },
+  { checkType: 'result', decisionCode: 'E72', decision: 'No match - CHED replaced' },
+  { checkType: 'result', decisionCode: 'E73', decision: 'No match - CHED deleted' },
+  { checkType: 'result', decisionCode: 'E75', decision: 'No match - Split consignment' },
+  { checkType: 'result', decisionCode: 'E87', decision: 'No match - Selected for HMI GMS inspection' },
+  { checkType: 'result', decisionCode: 'E84', decision: 'No match - Incorrect CHED type' },
+  { checkType: 'result', decisionCode: 'E99', decision: 'No match - Unknown error' },
+  { checkType: 'item', decisionCode: 'H01', decision: 'Hold - Decision not given' },
+  { checkType: 'item', decisionCode: 'H02', decision: 'Hold - To be inspected' },
+  { checkType: 'result', decisionCode: 'E88', decision: 'Hold - Awaiting IPAFFS update' },
+  { checkType: 'result', decisionCode: 'E74', decision: 'Hold - Partially rejected' },
+  { checkType: 'result', decisionCode: 'E85', decision: 'Hold - PHSI decision not provided' },
+  { checkType: 'result', decisionCode: 'E86', decision: 'Hold - HMI decision not provided' },
+  { checkType: 'item', decisionCode: 'C02', decision: 'Release - No inspection required' },
+  { checkType: 'item', decisionCode: 'C03', decision: 'Release - Inspection complete' },
+  { checkType: 'item', decisionCode: 'C05', decision: 'Release - Inspection complete temporary admission' },
+  { checkType: 'item', decisionCode: 'C06', decision: 'Release - Inspection complete T5 procedure' },
+  { checkType: 'item', decisionCode: 'C07', decision: 'Release - IUU inspection complete' },
+  { checkType: 'item', decisionCode: 'C08', decision: 'Release - IUU inspection not applicable' }
+]
+
+export const mapGmrCustomsDeclarations = ({
```

Comment:
move this function to `goods-vehicle-movements.js` and rename to `mapCustomsDeclarations`
The `getCustomsDeclarationStatus` can be exported and then consumed in `goods-vehicle-movements.js`